atlassian-labs/figma-for-jira
# NONE: Add UI and styling for the admin connect team flow

### Describe
Adding the screen for connecting a figma team. Upon success, it will render the success screen. Upon error, right now it will show the permissions error. We should update the `/admin/teams/:team_id/connect` route to return a 403 on when it fails to connect the team due to permissions errors and 500 for other errors when appropriate.

### Screenshots:
Connecting a team:
![image](https://github.com/atlassian-labs/figma-for-jira/assets/6136959/8feaa083-f1f8-4472-90d5-43926457b618)

When inputting an invalid URL:
![image](https://github.com/atlassian-labs/figma-for-jira/assets/6136959/e7346e99-3123-4fb4-bfd9-826a8a6021b8)


When there's an error:
![image](https://github.com/atlassian-labs/figma-for-jira/assets/6136959/2cd2c40b-08e5-418a-962d-d59597ff8e55)

Success:
![image](https://github.com/atlassian-labs/figma-for-jira/assets/6136959/3b4cc013-039b-4637-998f-d651c2f5838a)

### Notable changes:
- updated `/admin/teams/:team_id/connect` to return a teamSummary in the success case since we want to render the team name of the team we just connected in the success message

### Things to flag:
- I'm not sure if I need to be added to a feature flag for this to work but I was getting 415 errors with `await jiraService.setConfigurationStateInAppProperties()` which was recently added to `connect-figma-team-use-case.ts`. I just commented that part out when testing this flow

### Test Plan:
- open the configure page
- try to connect a random url
- see that it fails inline
- try to connect a figma team url you don't have admin permissions for
- see that it fails with the permissions section message
- try to connect a figma team url you do have the permissions for
- see that the success screen is rendered upon completion
- see that you can close this page with the `x`, `exit setup`, and `see connected screens`
